src/EnergyPlus/UnitVentilator.hh

```diff
@@ -67,127 +67,143 @@ struct EnergyPlusData;
 
 namespace UnitVentilator {
 
+    enum class CoilOption
+    {
+        Invalid = -1,
+        None,
+        Both,
+        Heating,
+        Cooling,
+        Num
+    };
+    enum class HeatCoilType
+    {
+        Invalid = -1,
+        Electric,
+        Gas,
+        Water,
+        Steam,
+        Num
+    };
+    enum class CoolCoilType
+    {
+        Invalid = -1,
+        Water,
+        Detailed,
+        HXAssisted,
+        Num
+    };
+    enum class OAControl
+    {
+        Invalid = -1,
+        VariablePercent,
+        FixedTemperature,
+        FixedOA,
+        Num
+    };
+
     struct UnitVentilatorData
     {
-        // Members
-        // Input data
         std::string Name;      // name of unit
         std::string SchedName; // availability schedule
```

Comment:
Should not need to store `SchedName` if you are immediately converting it to `SchedPtr`.